aospy_synthetic/test/test_db.py

```diff
@@ -1,43 +1,235 @@
 """Test suite for the aospy_synthetic db features."""
 
 import unittest
-from subprocess import call
+import os
 
-from test_objs import cases, models, example_projects, variables
-from aospy_synthetic.calc import Calc, CalcInterface
-from test_objs.SQLBackend.SQLAlchemyDB import SQLBackend
-#from aospy_synthetic.SQLAlchemyDB import SQLBackend
+from test_objs import (
```

Comment:
Relative imports here: `from .test_objs import ...` and `from . import heirarchical_test_objs ...` 

This will work once empty `__init__.py` created.